android/301Project/app/src/main/java/com/example/a301project/MealPlan.java

```diff
@@ -75,6 +78,34 @@ public String getId() {
         return this.id;
     }
 
+    /**
+     *
+     * @param startDate The start date in the format "yyyy-MM-dd"
+     * @param endDate The end date in the format "yyyy-MM-dd"
+     * @return Returns {@link Boolean} true if the start date is before (or equal to) the end date,
+     * false otherwise
+     */
+    public static boolean isStartDateBeforeEndDate(String startDate, String endDate) {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
+        Date sDate = null;
+        Date eDate = null;
+
+        try {
+            sDate = sdf.parse(startDate);
+            eDate = sdf.parse(endDate);
+
+            if (sDate.equals(eDate)) {
+                return true;
+            }
+            else {
+                return sDate.before(eDate);
+            }
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
```

Comment:
Since its Year, month, day theoretically a string comparison works too.
But who cares haha/

Reply:
Oh, haha, didn't even think of that